src/components/Create/components/pages/PayoutsPage/components/CreateFlowPayoutsTable.tsx

```diff
@@ -7,10 +7,10 @@ import {
   getV2V3CurrencyOption,
 } from 'packages/v2v3/utils/currency'
 import { MAX_DISTRIBUTION_LIMIT } from 'packages/v2v3/utils/math'
+import { allocationToSplit, splitToAllocation } from 'packages/v2v3/utils/splitToAllocation'
```

Comment:
Hmm... maybe we should move create into packages/v2v3 ...kinda makes sense actually

Reply:
You mean moving entire /create dir to v2v3?

Reply:
yeah dw for now. just an idea